xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -879,6 +879,18 @@ bool CPVRTimerInfoTag::HasEpgInfoTag(void) const
   return m_epgTag != NULL;
 }
 
+bool CPVRTimerInfoTag::HasSeriesEpgInfoTag(void) const
+{
+  if (m_epgTag &&
```

Comment:
don't trust that the system will evaluate conditions to bools? :)
`if (true) return true`